cpp/velox/compute/ArrowTypeUtils.cc

```diff
@@ -63,10 +69,14 @@ std::shared_ptr<arrow::DataType> toArrowType(const TypePtr& type) {
       return arrow::int32();
     case TypeKind::BIGINT:
       return arrow::int64();
+    case TypeKind::REAL:
+      return arrow::float32();
```

Comment:
Could you confirm this mapping again? Looks like Velox REAL type is C++ float and DOUBLE type is double.

```
template <>
struct CppToType<float> : public CppToTypeBase<TypeKind::REAL> {};

template <>
struct CppToType<double> : public CppToTypeBase<TypeKind::DOUBLE> {};
```

Reply:
There is a mapping table between Velox types and C++ types, https://facebookincubator.github.io/velox/develop/types.html.